### DISCUSSION TITLE
Updating records selectively during CSV import [Jun 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2569] [Discord Threads]
> 🙋‍♂️ **niroknox**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@990927987765563392> That's not possible from UI. You will have to write a script & use SDK / API to update records selectively.  
#  
> 🙋‍♂️ **niroknox**  
#### I see. are there any ready-made scripts or tools for that?  
#  
> 🙋‍♂️ **niroknox**  
#### I am on postgres - perhaps PGAdmin can do that?  
#  
> 💁 **o1lab**  
#### Have not tried to upload csv anytime  
#  
> 💁 **o1lab**  
#### https://learnsql.com/blog/how-to-import-csv-to-postgresql/  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/990928775967539210)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
